compiler/src/iree/compiler/Codegen/Common/GPU/GPUApplyTilingLevel.cpp

```diff
@@ -361,6 +361,13 @@ static LogicalResult applyTileAndFuseToEachRoot(
       return failure();
     }
 
+    if (IREE::Codegen::LoweringConfigAttrInterface originalConfig =
+            getLoweringConfig(tilingInterfaceOp)) {
+      if (!tiledResults->tiledAndFusedOps.empty()) {
+        setLoweringConfig(tiledResults->tiledAndFusedOps[0], originalConfig);
```

Comment:
Can we whitelist ops here for cases we know the tiled op is `[0]`? I'm guessing that is just `LinalgOp` and `AttentionOp`

Reply:
Actually never mind that sounds like a bad dependency to add.

Reply:
I guess nothing to addrress here, but the reason we have to do this is because the upstream implementation for LinalgOp drops the lowering_config (it's correct in doing that sadly...)